src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java

```diff
@@ -52,13 +57,17 @@ void checkRateOf(String accountId, RateLimiterKey key)
         }
     }
 
-    private Long updateAllowance(String key, int rateLimitInterval) {
+    private Long updateAllowance(String key, int rateLimitInterval) throws Exception {
         String derivedKey = getKeyForWindow(key, rateLimitInterval);
-        Long count = redisClientManager.getRedisConnection().sync().incr(derivedKey);
-        redisClientManager.getRedisConnection().sync().expire(derivedKey, rateLimitInterval / 1000);
+        Long count = time("redis.incr", () -> redisClientManager.getRedisConnection().sync().incr(derivedKey));
```

Comment:
Can we make the granularity of this metric clear  ? May be rename it to `redis.incr_[ns/nanoseconds]`

Reply:
Done.